roles/preflight-checks/tasks/ceph.yml

```diff
@@ -0,0 +1,26 @@
+---
+# this yml is executed with ceph installed
+- name: check ceph health
+  command: ceph -s
+  register: result_ceph_status
+  delegate_to: "{{ groups['ceph_monitors'][0] }}"
+
+- name: ceph status is not health ok
+  fail: msg="ceph status is not health ok"
+  when: not result_ceph_status.stdout |search("HEALTH_OK")
+
+- name: fail if no right cinder type
+  shell: . /root/stackrc; cinder type-list |grep CEPH
+  delegate_to: "{{ groups['controller'][0] }}"
+
+- name: check cinder.common_volume_name
+  fail: msg="cinder.common_volume_name is not true"
+  when: cinder.common_volume_name is defined and not cinder.common_volume_name | bool
+
+- name: check cinder configure
```

Comment:
Another change to name of the task: "Fail when cinder configuration has rbd_volumes"